src/main/java/uk/ac/ebi/ena/backend/service/FileDownloaderClient.java

```diff
@@ -183,17 +173,14 @@ public long downloadFTPUrlConnection(URL url, Path remoteFilePath, long size, in
             URLConnection conn = url.openConnection();
             long copied = 0;
             try (InputStream in =
-                         ProgressBar.wrap(conn.getInputStream(), getBProgressBar(fileName +
-                                 ":attempt " + (retryCount + 1), size));
+                         ProgressBar.wrap(conn.getInputStream(), getBProgressBar(fileName, size));
```

Comment:
you can still pass in the retry count from the fieldetail object and use that in the progressbar name